force_wfmanager/wfmanager_task.py

```diff
@@ -54,6 +57,12 @@ class WfManagerTask(Task):
     #: ZeroMQ Server to receive information from the running BDSS
     _zmq_server = Instance(ZMQServer)
 
+    #: Flag which says if the computation is running or not
+    _computation_running = Bool(False)
+
+    #: Flag which says if the menus should be disabled or not
+    _menu_enabled = Bool(True)
```

Comment:
Because TaskActions are enabled/disabled with `enabled_name` which is the name of the boolean trait which define if the action is enabled or not.